paper/paper.md

```diff
@@ -0,0 +1,445 @@
+23. Basic Stats III - Correlations
+24. Basic Stats IV - Significance
+25. Plotting I - Line Graphs
+26. Plotting II - Scatterplots
+27. Plotting III - Bar Charts and Histograms
+
+Each lesson builds on those before it, beginning with relevant content reminders
+from the previous lessons and ending with a concise summary of the skills
+presented within.
+Throughout our program, the students also simultaneously work on a data science
+project on a topic of interest using a real world dataset of their choosing.
+Through this Capstone Project, learners gain practical experience with each
+skill as they learn it in the lessons; including importing and cleaning data,
+data visualization, and basic statistical analyses.
+
+
+## Instructional Design
+<!-- teaching philosophy / pedagogy -->
+
+We modeled our instructional design in the style of Software Carpentry [@wilson_software_2016].
+
+1. Each lesson begins with a recapping of the relevant core skills presented in the previous lessons.
+1. All lessons are designed to be taught via 15-minute live-coding sessions, where students type and run code in their own notebooks along with the instructor in real time. As in Software Carpentry, we find this to be a highly effective method of teaching coding, since students must actively engage with the material and deal with errors and bugs as they come up.
+1. Each lesson ends with a summary of core skills presented within the material.
+1. Each short lesson is also accompanied by a subsequent 10-minute independent practice, providing further opportunity for practical experience implementing the coding skill at hand and testing their understanding of the content.
+
+This curriculum was originally developed for in-person instruction, but the onset of the COVID-19 pandemic necessitated restructuring to a virtual format.
+To better facilitate virtual instruction, we switched to a flipped classroom style based on feedback from our club participants that it was too difficult to follow along with live coding via Zoom.
+We provide video recordings of instructors going through the Jupyter notebooks as if they were live coding in the classroom.
+Students then watch the lessons and complete the practice exercises prior to virtual meetings, in which they have the opportunity to ask questions on material they did not understand and go over the practice exercises.
+This virtual format is especially beneficial because it 1) allows students to learn at their own pace, and 2) enables dissemination of our curriculum to a wider audience interested in learning introductory Python programming for data science.
+
+For both in-person and virtual instruction, once students have completed the Fundamentals module and reach the Data Science Essentials module they will begin simultaneous work on their data science projects.
+Projects are completed in a paired programming style, where partners take turns assuming the “driver” (i.e. the typer) and “navigator” (i.e. the instructor) roles.
```

Comment:
I think an additional sentence or two more clearly elaborating on how paired programming works (something like "The navigator proposes a solution to the problem while the driver physically types the solution") might be useful, just to further clarify the two roles for those who aren't familiar with paired programming

Reply:
Could also/instead cite a blog post or paper that describes it in more detail.